nflow-engine/src/main/resources/scripts/db/update-7.2.5-x/oracle.update.ddl.sql

```diff
@@ -0,0 +1,17 @@
+alter sequence nflow_executor_id_seq nocache
+/
+
+alter sequence nflow_workflow_id_seq nocache
+/
+
+alter sequence nflow_workflow_action_id_seq nocache
+/
+
+create or replace trigger nflow_workflow_action_insert
+  before insert on nflow_workflow_action
+  for each row
+declare
+begin
+  :new.id := nflow_workflow_action_id_seq.nextval;
+end;
+/
```

Comment:
Perhaps set nflow_workflow_action_id_seq to nflow_workflow_id_seq current value after replacing the trigger, so that there's minimal chance of colliding ids.

Reply:
Or action_seq=workflow_seq.nextval, as calling currentval requires calling nextval in the same session (if I remember my Oracle correctly from year 2004)

Reply:
Changed to recreate the sequence using `max(id) from nflow_workflow_action + 1000` **before** replacing the trigger. That way the trigger should generate unique ids immediately after it's replaced.